crates/db/migrations/20251005000000_update_shared_tasks_nullable.sql

```diff
@@ -0,0 +1,97 @@
+--
+-- Migration steps following the official SQLite "12-step generalized ALTER TABLE" procedure:
+-- https://www.sqlite.org/lang_altertable.html#otheralter
+--
+PRAGMA foreign_keys = OFF;
+
+-- This is a sqlx workaround to enable BEGIN TRANSACTION in this migration, until `-- no-transaction` lands in sqlx-sqlite.
+-- https://github.com/launchbadge/sqlx/issues/2085#issuecomment-1499859906
+COMMIT TRANSACTION;
```

Comment:
Let's just squash this now - there's nobody using it with a prod db. Can squash in the `unique_shared_task_index.sql` too